spotipy/cache_handler.py

```diff
@@ -138,3 +145,32 @@ def save_token_to_cache(self, token_info):
             self.request.session['token_info'] = token_info
         except Exception as e:
             logger.warning("Error saving token to cache: " + str(e))
+
+
+class RedisCacheHandler(CacheHandler):
+    """
+    A cache handler that stores the token info in the Redis.
+    """
+
+    def __init__(self, redis):
+        """
+        Parameters:
+            * redis: Redis object provided by redis-py library
+            (https://github.com/redis/redis-py)
+        """
+        self.redis = redis
+
+    def get_cached_token(self):
+        token_info = None
+        try:
+            token_info = json.loads(self.redis.get('token_info'))
```

Comment:
Won't `self.redis.get('token_info')` be `None` if the value hasn't yet been stored in Redis and then `json.loads(None)` throw an exception?

Reply:
@Findus23 
You're right. I'm sorry for that. I'll fix it. 
Should I create a new branch and new pull request?